lib/auth/clt.go

```diff
@@ -304,11 +304,7 @@ func (c *Client) CreateCertAuthority(ca types.CertAuthority) error {
 
 // RotateCertAuthority starts or restarts certificate authority rotation process.
 func (c *Client) RotateCertAuthority(ctx context.Context, req RotateRequest) error {
-	caType := "all"
-	if req.Type != "" {
-		caType = string(req.Type)
-	}
-	_, err := c.PostJSON(ctx, c.Endpoint("authorities", caType, "rotate"), req)
+	_, err := c.PostJSON(ctx, c.Endpoint("authorities", string(req.Type), "rotate"), req)
```

Comment:
Should we check for a missing type here too? What happens if you call this without specifying a type in the request?

Reply:
I did notice that this wasn't being used but when I'm working on changes that will be backported I tend to try and not making bigger changes than necessary.